### PR TITLE
Film details page

### PR DESCRIPTION
**Pull request by dsmith2708**
added film details page and film object, refactored new releases and all films pages

### CODE REVIEW

File: app/views/filmDetails.scala.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <div class="col-4">
+                <img width="150" height="200" src="@film.getPosterUrl()">
+            </div>
+            <div class="col-8">
+                <p>@film.getDescription()</p>
+            </div>
+        </div>
+        <div class="film-release-credits-booking-container row">
+            <div class="col-4">
+                <!-- film release date -->
```
Comment (tvaidotas) [inline app/views/filmDetails.scala.html]: do you need these comments?

File: app/views/index.scala.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 @main("QA Cinema") {
     <h1>Welcome to QA Cinemas</h1>
 
-    @newRelesesGallery()
+    @newRelesesGallery(PlaceholderDatabase.allFilmsList)
```
Comment (tvaidotas) [inline app/views/index.scala.html:6]: Does this actually work? I would expect that an import would be required before it works

**Review by tvaidotas — COMMENTED** [review]
just fix those couple of things pls

**Comment by dsmith2708** [comment]
fixes made to remove unnecessary default values and comments. code in index.scala.html to pass film list to new releases works and no changes are required there

**Pull request merged**